helm_deploy/templates/deployment.yaml

```diff
@@ -19,7 +20,7 @@ spec:
       {{- end }}
       labels:
         {{- include "laa-submit-crime-forms.selectorLabels" . | nindent 8 }}
-        app: web
+        app: {{ include "laa-submit-crime-forms.fullname" . }}
```

Comment:
my preference would be to use the file name instead of the label to differentiate as the names behind these variables already communicate what they are: 
laa-submit-crime-forms-app
laa-submit-crime-forms-app-worker

Reply:
we could maybe name the deployment.yaml file deployment-app or deployment-web?

Reply:
No need to do this